### PR TITLE
All in one solution 

### PR DESCRIPTION
**Pull request by JohnnyMaxK**
https://vlingoplatfor-i6o3730.slack.com/archives/CQDJXDX2L/p1579768025000300

### CODE REVIEW

**Comment by tjaskula** [comment]
#1

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+test_script:
+- dotnet test all_in_one.sln
+deploy:
+- provider: NuGet
+  api_key:
+    secure: S7SGXqFTXTOVbVLP45Ofyf/iTsIOr9o0F/QEeQkaUMqb/PkwWjXK6+CF4WAqbfWr
+  skip_symbols: true
+  artifact: /.*\.nupkg/
+  on:
+    branch: master
```
Comment (tjaskula) [inline appveyor.yml:22]: So potentially I could use that to publish all the NuGet packages at once ? I guess I would need to change `Directory.Build.props` for every submodule. I'm not sure how the publishing to NuGet works then.

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      git checkout -qf $env:APPVEYOR_REPO_COMMIT
+    } else {
+      git clone -q https://github.com/$env:APPVEYOR_REPO_NAME.git $env:APPVEYOR_BUILD_FOLDER
+      cd $env:APPVEYOR_BUILD_FOLDER
+      git fetch -q origin +refs/pull/$env:APPVEYOR_PULL_REQUEST_NUMBER/merge:
+      git checkout -qf FETCH_HEAD
+    }
+    if( $isLinux ) {
+      sudo ln -s /usr/local/share/dotnet/dotnet /usr/local/bin/
+    }
```
Comment (tjaskula) [inline appveyor.yml:44]: If lines before are specific to Appveyor do we need that ?

**Review by tjaskula — COMMENTED** [review]
Great job @Johnny-Bee.

Waiting just for your answers before pushing the button :)

File: appveyor.yml (hunk 1 above)
Comment (JohnnyMaxK) [reply appveyor.yml:22]: Theoretically yes, but to be tested! You will have to change the `Directory.Build.props` on every submodule. We can put a condition on the project file to take into account another property called, for example, `GlobalVlingoVersion` that will be declared on the platform scope that can override the submodule property `VlingoVersion`.

File: appveyor.yml (hunk 2 above)
Comment (JohnnyMaxK) [reply appveyor.yml:44]: I'm sorry, I don't know. Must be tested.

**Review by tjaskula — APPROVED** [review]

File: appveyor.yml (hunk 1 above)
Comment (JohnnyMaxK) [reply appveyor.yml:22]: After long testing, it's possible what I said but I think it was a wrong idea to put a `Directory.Build.props` on the platform repository. Each repository must keep the version used to publish to keep track of the change.

**Pull request merged**